app/views/petitions/search/result_items/_petition_result_for_facet_collecting_signatures.html.erb

```diff
@@ -0,0 +1,2 @@
+<h2><%= link_to petition.action, petition_path(petition) %></h2>
+<p><%= signature_count(:default, petition.signature_count) %></p>
```

Comment:
Might as well leave out signature counts from the lists

Reply:
Sure, but why?

Reply:
Nope, gotcha

Reply:
Found it

Reply:
will do all of these in a subsequent PR addressing the specific Trello card asking for it